manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -375,6 +381,8 @@ jobs:
             uaa: {}
       metron_agent:
         zone: z1
+      route_registrar:
+        routes: []
```

Comment:
Is there any way we can just pull in the `common` package directly instead of having to have this somewhat hacky workaround.

This also sounds like an issue that should be reported upstream. It seems very surprising that `statsd-injector` breaks when `route_registrar` isn't present.